cves/CVE-2017-12616.yml

```diff
@@ -44,20 +44,19 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  The use of the class VirtualDirContext, which is intended to expose
-  multiple directories of the filesystem under a single webapp, made it possible for
-  an attacker to insert a specially crafted request that would enable them to bypass
-  security constraints and view the source code for JavaServer Pages.
+  A specially crafted request made it possible for users to gain access to
+  sensitive information such as source code, using the class VirualDirContext.
```

Comment:
Very clear as to what the vulnerability was, this could be a good place to talk about the fix that was implemented to solve this vulnerability.

Reply:
Added a brief description as to how they solved it.